backend/cart-service/lambda/src/main.ts

```diff
@@ -0,0 +1,34 @@
+import { NestFactory } from '@nestjs/core';
+import serverlessExpress from '@codegenie/serverless-express';
+import helmet from 'helmet';
+import { Callback, Context, Handler } from 'aws-lambda';
+import { AppModule } from './app.module';
+
+const port = process.env.PORT || 4000;
+let server: Handler;
+
+async function bootstrap() {
+    const app = await NestFactory.create(AppModule);
+
+    app.enableCors({
+        origin: (req, callback) => callback(null, true),
+    });
+    app.use(helmet());
+
+    await app.listen(port);
```

Comment:
Nest.js application should NOT listen if it's used in Lambda.

Reply:
sounds reasonable. I just followed the official guide https://docs.nestjs.com/faq/serverless#benchmarks
with the following:
```js
// #2 Nest (with @nestjs/platform-express)
import { NestFactory } from '@nestjs/core';
import { AppModule } from './app.module';

async function bootstrap() {
  const app = await NestFactory.create(AppModule, { logger: ['error'] });
  await app.listen(3000);
}
bootstrap();
```